Misc test fixes

bshaffer opened the pull request.

tmatsuo commented on error_reporting/composer.json:
```diff
@@ -7,5 +7,10 @@
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Google\\Cloud\\Samples\\Monitoring\\": "test/lib"
```
The directory is `error_reporting` while the namespace is `Monitoring`?

bshaffer replied:
Good catch! These should all be fixed now.

tmatsuo commented on error_reporting/test/error_reportingTest.php:
```diff
@@ -17,19 +17,13 @@
 
 namespace Google\Cloud\Samples\Monitoring;
```
Should we match the namespace to the directory?

tmatsuo commented on error_reporting/test/quickstartTest.php:
```diff
@@ -15,17 +15,13 @@
-use GuzzleHttp\Client;
-use GuzzleHttp\HandlerStack;
-use Google\Cloud\TestUtils\EventuallyConsistentTestTrait;
+namespace Google\Cloud\Samples\Monitoring;
```
ditto

tmatsuo commented:
The code looks good.

Do you know why one of the tests in logging was skipped?
https://travis-ci.org/GoogleCloudPlatform/php-docs-samples/jobs/337274790#L4223

bshaffer commented:
@tmatsuo amazing!! Yes, I renamed a test from `testWrite` to `testWriteAndList` and forgot to update the next test which had `@depends testWrite`. This is now fixed.
